include/Decimal.hpp

```diff
@@ -53,6 +53,10 @@ class Decimal {
 
     Decimal() {}
 
+    Decimal(const TTInt& ttInt) {
+        getDecimal() = ttInt;
```

Comment:
Can use assignment also. The getDecimal() already is doing the same, so reused the functionality.